### DISCUSSION TITLE
JavaScript SQL alternative that works with GitHub

### DISCUSSION THREAD

**Discussion by ghost** [community/community#69363] [Programming Help] [answered]
### Body

Hi,

I am building a website that allows students from HWDSB to get information about work, submit work, get teacher feedback and finally get notified.

I wanted to know any JavaScript database I could use to achieve this

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebubekirgungor** [comment, marked as answer]
Hello, I did not understand what javascript database is, but you can easily use MongoDB with javascript (node.js). MongoDB is a NoSQL database.
By the way, I have answered your two questions last week, I would be happy if you mark them as answered.
